core/textinput/src/textinput/TextInput.cpp

```diff
@@ -338,6 +338,16 @@ namespace textinput {
                   [](Display *D) { return D->NotifyWindowChange(); });
   }
 
+  void TextInput::SetMaxDepth(size_t maxDepth) const
```

Comment:
Given that those apply to history, I would suggest renaming (same for `SetPruneLength()`), as in
```suggestion
  void TextInput::SetHistoryMaxDepth(size_t maxDepth) const
```

Reply:
Done.